sdk/communication/communication-phone-numbers/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 1.2.1 (Unreleased)
 
 ### Features Added
+- Added support for API version `2023-10-01-preview`, This gives us functionality for Do Not Resell Consent Changes
+- Users will now be able to purchase from countries like `IT`, `AT`, `FR`, `NO`, `PT` by agreeing not to resell phone numbers by setting consentToNotResellNumbers to true
```

Comment:
Users will be able to purchase from the countries 'IT', 'AT', 'FR', 'NO, 'PT'. They can do so by agreeing ...